projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-flag/components/case-flag-summary-list/case-flag-summary-list.component.html

```diff
@@ -19,33 +20,67 @@
     <dd class="govuk-summary-list__value">
       {{flagDescription}}
     </dd>
-    <dd class="govuk-summary-list__actions" *ngIf="summaryListDisplayMode === displayMode.CREATE">
-      <a class="govuk-link" href="javascript:void(0)" (click)="changeButtonEmitter.emit(caseFlagFieldState.FLAG_TYPE)">
+    <dd class="govuk-summary-list__actions">
+      <a class="govuk-link" href="javascript:void(0)"
+        (click)="changeButtonEmitter.emit(this.summaryListDisplayMode === displayMode.MANAGE ? caseFlagFieldState.FLAG_MANAGE_CASE_FLAGS : caseFlagFieldState.FLAG_TYPE)">
         Change<span class="govuk-visually-hidden"> description</span>
       </a>
     </dd>
   </div>
-  <div class="govuk-summary-list__row">
+  <div class="govuk-summary-list__row" *ngIf="flagDescriptionWelsh?.length > 0">
+    <dt class="govuk-summary-list__key">
+      Other description (Welsh)
+    </dt>
+    <dd class="govuk-summary-list__value">
+      {{flagDescriptionWelsh}}
+    </dd>
+    <dd class="govuk-summary-list__actions">
+      <a class="govuk-link" href="javascript:void(0)"
+        (click)="changeButtonEmitter.emit(this.summaryListDisplayMode === displayMode.MANAGE ? caseFlagFieldState.FLAG_UPDATE_WELSH_TRANSLATION : caseFlagFieldState.FLAG_TYPE)">
+        Change<span class="govuk-visually-hidden"> description</span>
```

Comment:
@johnbenjamin-hmcts - this should be `Change<span class="govuk-visually-hidden"> description (Welsh)</span>` otherwise the screen reader will read out "Change description" twice

Reply:
Thank you @Daniel-Lam, I have updated the code.